### PR TITLE
Use conda-build-all for circle CI builds

### PR DESCRIPTION
**Pull request by dkillick**
Use `conda-build-all` to run conda recipes builds on Circle CI. See also #190.

### CODE REVIEW

**Comment by dkillick** [comment]
Looks like this one is good to go too!

File: circle.yml [hunk 1]
```diff
@@ -6,8 +6,7 @@ dependencies:
   # Note, we used to use the naive caching of docker images, but found that it was quicker
   # just to pull each time. #rollondockercaching
   override:
-    - docker pull pelson/obvious-ci:latest_x64
-#    - docker pull pelson/conda32_obvious_ci
+    - docker pull condaforge/linux-anvil
```
Comment (pelson) [inline circle.yml]: Everything 👍  except this.

Comment (dkillick) [inline circle.yml]: To add some meat to the above comment's bones: the condaforge docker image is a lot more sophisticated (and newer) than the obvious-ci docker image. The condaforge image also takes a number of steps to try and preserve backward-compatibility to ~RHEL6. Overall though, while newer is good, we probably don't need the extra complexity of the condaforge docker image in conda-recipes-scitools, especially if it causes some unexpected impacts.

Comment (dkillick) [inline circle.yml]: So basically, I'm going to revert this change.

**Comment by pelson** [comment]
Thanks @dkillick!
